package.json

```diff
@@ -134,8 +134,5 @@
       "not samsung <= 4"
     ],
     "node": "16"
-  },
-  "engines": {
```

Comment:
Curious why we need remove this

Reply:
We don't 'need' to, but I figured no reason to limit it to node 18. Have to keep switching versions back and forth locally. No reason it won't work with higher node versions